### PR TITLE
Desctop version

### PR DESCRIPTION
**Pull request by sajibarafatsiddiqui**
1) Added linter config file.
2) Using media queries made the mobile fast and compatiable with the desktop.
3) Followed git hub flows.

### CODE REVIEW

**Review by Ugboaja-Uchechi — COMMENTED** [review]
Hi @sajibarafatsiddiqui, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't have all the headers as seen in the [template](https://github.com/microverseinc/readme-template) provided by Microverse. Kindly add all the headers and section to your Readme file 🙏 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: sajibportfolio.css [hunk 1]
```diff
@@ -1234,3 +1221,352 @@ button:disabled {
   color: #fff;
   background-color: #6070ff;
 }
+
+@media (min-width: 1024px) {
```
Comment (Favourezeugwa) [inline sajibportfolio.css:1225]: - There is currently excess whitespace on the body of your desktop version when i scroll horizontally to the right direction. Can we fix this as well?

Here is what I mean 👇🏽 

exps

**Review by Favourezeugwa — CHANGES_REQUESTED** [review]
## STATUS: REQUIRES CHANGES 🚦 ⚠️ ♻️ 

Hi @sajibarafatsiddiqui  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close-brianna


## Highlights 💯 
- No Linter errors ✔️
- Descriptive Pull request summary✔️
- Good commit messages ✔️
- Professional Readme file ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: sajibportfolio.css (hunk 1 above)
Comment (sajibarafatsiddiqui) [reply sajibportfolio.css]: image

solved!

File: sajibportfolio.css [hunk 2]
```diff
@@ -1234,3 +1221,352 @@ button:disabled {
...
+  .home-page {
+    display: inline-flex;
+    flex-direction: column;
+    align-items: flex-start;
+    padding: 24px;
+    height: 500px;
+    border-radius: 0 0 0 5%;
+    gap: 20px;
+    padding-top: 0;
+  }
```
[the remark this answers is not in the document]
Comment (sajibarafatsiddiqui) [reply sajibportfolio.css]: image

solved!

File: sajibportfolio.css [hunk 3]
```diff
@@ -1234,3 +1221,352 @@ button:disabled {
...
+    padding-right: 60px;
+    margin-top: 0;
+    place-items: center;
+    width: 93%;
+    display: grid;
+    grid-row-start: initial;
+    grid-template-columns: 40% 60%;
+    grid-template-rows: max-content;
+  }
+
```
[the remark this answers is not in the document]
Comment (sajibarafatsiddiqui) [reply sajibportfolio.css]: image

Solved!

File: sajibportfolio.css [hunk 4]
```diff
@@ -1234,3 +1221,352 @@ button:disabled {
...
+    gap: 40px;
+    height: 550px;
+    left: 0;
+    border: 1px solid #dfe1e6;
+    border-radius: 16px;
+    width: 80%;
+    margin-left: 10%;
+    grid-column-gap: 10px;
+  }
+
```
[the remark this answers is not in the document]
Comment (sajibarafatsiddiqui) [reply sajibportfolio.css]: image

solved!

**Comment by sajibarafatsiddiqui** [comment]
My collaborator coding friend mistakenly made a commit in main on solving the animation project.

**Review by amejid — COMMENTED** [review]
Hi @sajibarafatsiddiqui, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that linter checks are not running.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: sajibportfolio.html [hunk 5]
```diff
@@ -1,289 +1,294 @@
...
+            <!-- <img class="counter" src="statics/images/Counter.png" alt="counter"/> -->
+            <div class="jt">
+              <p class="jtt">Back End Dev</p>
+            </div>
+            <span class="ct1"></span>
+            <!--<img class="counter" src="statics/images/Counter.png" alt="counter">-->
+            <div class="yr">
+              <p class="yrt">2015</p>
             </div>
           </div>
```
Comment (PinkMoon25) [inline sajibportfolio.html:156]: - [ ] As requested by the previous reviewer, Please make sure the project card follows the figma template as close as possible, also your 3rd and 4th project cards does not have the images displayed see :point_down: 
image

currently you have excessive hap between project images and project details, how It is :point_down: 
image

how it should be :point_down: 
image

you have morespace inbetween and no space on left and right side.

File: sajibportfolio.html [hunk 6]
```diff
@@ -1,289 +1,294 @@
...
+            <a href="#" target="_blank"><img src="statics/images/doits.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/twitter.png" alt="angel list link" /></a>
+          </li>
+          <li class="nb1">
+            <a href="#" target="_blank"><img src="statics/images/M.png" alt="angel list link" /></a>
+          </li>
+        </ul>
+        <button class="rsm"><a class="brt" target="_blank" href="#">Get my resume</a></button>
```
Comment (PinkMoon25) [inline sajibportfolio.html:254]: - [ ] As requested earlier. please implement the hover functionality for all your buttons, see design :point_down: 
image

File: sajibportfolio.html [hunk 7]
```diff
@@ -1,289 +1,294 @@
...
-            
-         
+        <li>
+          <div id="spec_name">
+            <p class="lfs">Languages</p>
+            <img class="lfsi" src="statics/images/fleche_dessous.png">
+          </div>
+
+          <div class="programmes">
+            <ul>
```
Comment (PinkMoon25) [inline sajibportfolio.html:269]: - [ ] As requested, Please fix the language skills section, currently the alignment of skills is not correct, your design :point_down: 
image

How it should be :point_down: 
image

**Review by PinkMoon25 — CHANGES_REQUESTED** [review]
Hi @sajibarafatsiddiqui , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.



 ## Highlights
linters checks :heavy_check_mark: 
Professional readme :heavy_check_mark: 
used flex and grid for layout :heavy_check_mark: 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

[OPTIONAL] Please consider using meaningful classnames, currently you have class='rsa' class='fra' , It will help anybody reading the code understand it clearly and will also help in debugging, improves code readability :pray: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: sajibportfolio.html (hunk 6 above)
Comment (sajibarafatsiddiqui) [reply sajibportfolio.html:246]: applied. Thanks

File: sajibportfolio.html (hunk 7 above)
Comment (sajibarafatsiddiqui) [reply sajibportfolio.html:261]: applied. Thanks

File: sajibportfolio.html (hunk 5 above)
Comment (sajibarafatsiddiqui) [reply sajibportfolio.html]: removed the space in between thanks.

Comment (sajibarafatsiddiqui) [reply sajibportfolio.html]: removed comments from html

Comment (sajibarafatsiddiqui) [reply sajibportfolio.html]: fixed changing the image url which was relative for card 3 and 4

**Pull request merged**